workspace.spec

```diff
@@ -572,6 +572,378 @@ module Workspace {
+				"Datacite:IsSourceOf"
+				"Datacite:IsSupplementTo"
+				"Datacite:IsSupplementedBy"
+				"Datacite:IsVariantFormOf"
+				"Datacite:IsVersionOf"
+				"Datacite:Obsoletes"
+				"Datacite:References"
+				"Datacite:Requires"
+				"Datacite:Reviews"
+				"CrossRef:BasedOnData"
+				"CrossRef:Finances"
+				"CrossRef:HasComment"
+				"CrossRef:HasDerivation"
+				"CrossRef:HasExpression"
+				"CrossRef:HasFormat"
+				"CrossRef:HasManifestation"
+				"CrossRef:HasManuscript"
+				"CrossRef:HasPreprint"
+				"CrossRef:HasRelatedMaterial"
+				"CrossRef:HasReply"
+				"CrossRef:HasReview"
+				"CrossRef:HasTranslation"
+				"CrossRef:IsBasedOn"
+				"CrossRef:IsBasisFor"
+				"CrossRef:IsCommentOn"
+				"CrossRef:IsDataBasisFor"
+				"CrossRef:IsExpressionOf"
+				"CrossRef:IsFinancedBy"
+				"CrossRef:IsFormatOf"
+				"CrossRef:IsManifestationOf"
+				"CrossRef:IsManuscriptOf"
+				"CrossRef:IsPreprintOf"
+				"CrossRef:IsRelatedMaterial"
+				"CrossRef:IsReplacedBy"
+				"CrossRef:IsReplyTo"
+				"CrossRef:IsReviewOf"
+				"CrossRef:IsSameAs"
+				"CrossRef:IsTranslationOf"
+				"CrossRef:Replaces"
+	*/
+	typedef structure {
+		string id;
+		string description;
+		string relationship_type;
+	} PermanentID;
+
+	/* Title
+		Represents the title or name of a resource.
+
+		The 'title_string' field is required; if no value is supplied for 'title_type', it
+		defaults to 'title'. If the 'title_type' is set to 'translated_title', the appropriate
+		BCP-47 tag must be supplied in the 'title_language' field.
+
+		string title_string - the resource title
+		string title_language (optional) - language that the title is in, as a IETF BCP-47 tag.
+			Examples: 'fr'; 'jp-JP'; 'zh-Hant-CN'; 'en-Latn-GB'; 'mn-Cyrl'.
+		string title_type (optional) - the type of title described; must be one of the following:
+			"title" (default)
+			"subtitle"
+			"alternative_title"
+			"translated_title"
+			"other"
+	*/
+	typedef structure {
+		string title_string;
+		string title_language;
+		string title_type;
+	} Title;
+
+	/* Contributor
+
+		Represents a contributor to the resource.
+
+		Contributors must have a 'contributor_type', either 'person' or 'organization'.
+
+		There must also be an identifier or name of some sort for the contributor.
+
+		If the contributor is an organization, either the 'contributor_id' or
+		the 'name' field must be populated.
+
+		An individual can be represented by either 'given_name' and 'family_name'; a 'name'
+		(if the name is not in the binomial format); or a `contributor_id`, such as an
+		ORCID.
+
+		The 'contributor_role' field takes values from the Datacite and CRediT contributor
+		roles vocabularies. For more information on these resources and choosing the
+		appropriate roles, please see the following links:
+
+		Datacite contributor roles: https://support.datacite.org/docs/datacite-metadata-schema-v44-recommended-and-optional-properties#7a-contributortype
+
+		CRediT contributor role taxonomy: https://credit.niso.org
```

Comment:
It'd be nice if all these orgs with overlapping categories could work together

Reply:
I'm going to create a new controlled vocabulary that unites these two efforts and is the new standard CV for contributor roles.

Reply:
Are you serious or kidding? I think you're kidding but if you are serious I wouldn't do that